### PR TITLE
Feat/image demo broken

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
ImageScreen - Adding demo for Image's 'errorSource' prop.

## Changelog
ImageScreen - Adding demo for Image's 'errorSource' prop.

### CODE REVIEW

File: demo/src/screens/componentScreens/ImageScreen.tsx [hunk 1]
```diff
@@ -1,19 +1,19 @@
 import React, {Component} from 'react';
-import {View, Text, Image, Colors} from 'react-native-ui-lib';
+import {View, Text, Image, Colors, Assets} from 'react-native-ui-lib';
 import {renderBooleanOption, renderRadioGroup, renderSliderOption} from '../ExampleScreenPresenter';
 
-import cameraIcon from '../../assets/icons/cameraSelected.png';
```
Comment (ethanshar) [inline demo/src/screens/componentScreens/ImageScreen.tsx:5]: This is causing an issue when enabling the "Show Overlay Content" toggle.

File: demo/src/screens/componentScreens/ImageScreen.tsx [hunk 2]
```diff
@@ -63,6 +64,7 @@ class ImageScreen extends Component {
               {renderBooleanOption.call(this, 'Show Overlay Content', 'showOverlayContent')}
               {renderRadioGroup.call(this, 'Overlay Type', 'overlayType', {none: 'none', ...Image.overlayTypes})}
               {renderSliderOption.call(this, 'Margin(margin-XX)', 'margin', {step: 4, min: 0, max: 40})}
+              {renderBooleanOption.call(this, 'Show Error Image', 'showErrorImage')}
```
Comment (ethanshar) [inline demo/src/screens/componentScreens/ImageScreen.tsx:67]: Move the Switch up together with the other toggles

File: demo/src/screens/componentScreens/ImageScreen.tsx (hunk 1 above)
Comment (Inbal-Tish) [reply demo/src/screens/componentScreens/ImageScreen.tsx:5]: Fixed

File: demo/src/screens/componentScreens/ImageScreen.tsx (hunk 2 above)
Comment (Inbal-Tish) [reply demo/src/screens/componentScreens/ImageScreen.tsx]: Done

**Review by ethanshar — APPROVED** [review]

**Pull request merged**